client/assets/translations/translation.csv

```diff
@@ -1,36 +1,36 @@
 keys,en,de,fr
-REFILLED,Refilled,aufgefüllt,rempli
-ENTERED,Entered,eingestiegen,entré
-DEPLOYED,Deployed,abgesetzt,abandonné
-JOINED,Joined,fusioniert,fusionné
+REFILLED,Refilled,aufgef?llt,rempli
+ENTERED,Entered,eingestiegen,entr?
+DEPLOYED,Deployed,abgesetzt,abandonn?
+JOINED,Joined,fusioniert,fusionn?
 ,,,
 TUTORIAL,Tutorial,Tutorial,Didacticiel
 CAMPAIGN,Campaign,Kampagne,campagne
 VS_AI,VS AI,VS KI,contre IA
 VS_PLAYER,VS Player,VS Spieler,contre joueur
 START,Start,Start,commencer
-BACK,Back,Zurück,revenir
+BACK,Back,Zur?ck,revenir
```

Comment:
hey! be careful, or is github fucking it up? 

Which letter codification we have in the csv?  (and which one is required by godot?)

Reply:
Damn. I'm guessing I made a mistake with libre office.